plugins/qcheck-stm/src/ir_of_gospel.ml

```diff
@@ -383,8 +377,8 @@ let ghost_functions =
 let run sigs config =
   let open Reserr in
   let open Ir in
-  let* state = state config sigs in
+  let* state, invariants = state config sigs in
```

Comment:
This line reads a bit funny. I suggest renaming the `state` function as `state_and_invariants`.